website/frontend/src/components/blocks/UploadMeta/UploadMeta.tsx

```diff
@@ -0,0 +1,84 @@
+import { ChangeEvent, useState } from 'react';
+import { Metadata, getWasmMetadata } from '@gear-js/api';
+import { FileInput } from '@gear-js/ui';
+
+import styles from './UploadMeta.module.scss';
+import { UploadData } from './types';
+import { getMetaProperties } from './helpers';
+
+import { useAlert } from 'hooks';
+import { readFileAsync, checkFileFormat } from 'helpers';
+import { formStyles, FormText } from 'components/common/Form';
+
+type Props = {
```

Comment:
I suggest to store **Props** in the local types.ts.

Reply:
> I suggest to store **Props** in the local types.ts.

why tho? we declare `Props` in the same component file everywhere, unless it's too complicated, and I see no reason to do otherwise

Reply:
When we create a new component and write some CSS. We can write only one rule but we store it separately. One type is the same thing and as for me better to store it separately too.

Reply:
We have a different approach everywhere. And the example with css is generally incorrect, since it is a different language.